packages/app/src/app/overmind/namespaces/editor/actions.ts

```diff
@@ -120,11 +120,7 @@ export const sandboxChanged: AsyncAction<{ id: string }> = withLoadApp<{
     state.editor.modulesByPath = fs;
   });
 
-  if (
-    sandbox.featureFlags &&
-    sandbox.featureFlags.containerLsp &&
-    !sandbox.owned
-  ) {
+  if (sandbox.featureFlags.containerLsp && !sandbox.owned) {
```

Comment:
I think you need to get it from local storage here :/

Reply:
@christianalfoni added a `featureFlags` and was checking for it in this please, so I just kept it here

```diff
- sandbox.featureFlags &&
  sandbox.featureFlags.containerLsp &&
```

Reply:
But that's on the types, I am not sure it's used, it would be best to get the feature flag